en/index.md

```diff
@@ -322,6 +323,118 @@ permitting users or customers to conduct certain activities.
 It can be tempting to use flowery, official-sounding words
 rather than plain English. Try to keep it simple.
 
+## Interacting with UI elements
+
+### **Screenshots**
+
+**Screenshots should be used sparingly**
+
+A screenshot is not a replacement for clear descriptions in documentation. If an image is well described, a screenshot shouldn’t be necessary in many situations, and including many screenshots can clutter the documentation.
+
+Screenshots also can’t be translated, so they aren’t as accessible to non-native English users or those using translated documentation. Additionally, those using screen-readers won’t be able to access the screenshots without alt-text.
+
+### **Using UI elements as the English words**
+
+**Don’t use UI elements like they’re English words**
+
+For example:
+
+- **Use**: When you’re finished, click **Save** to save your settings.
+- **Don’t use**: When you’re finished, **Save** your settings.
+
+Using the UI text as English words is less clear and may not transfer well to certain translated versions.
+
+### **Click vs. Tap vs. Select**
+
+**Use “Click” for buttons that you click on (or “Tap” if the product is primarily for mobile devices)**
```

Comment:
I don't think all this bold is necessary and this probably works better as a list
```suggestion
 - Use “Click” for buttons that you click on (or “Tap” if the product is primarily for mobile devices)
```